torch/accelerator/__init__.py

```diff
@@ -139,7 +140,7 @@ def set_device_index(device: _device_t, /) -> None:
 )(set_device_index)
 
 
-def current_stream(device: _device_t = None, /) -> torch.Stream:
+def current_stream(device: _device_t = None) -> torch.Stream:
```

Comment:
```suggestion
def current_stream(device: _device_t = None, /) -> torch.Stream:
```

Reply:
Do we really not want user to pass this one as a kwarg?